src/components/BigAssButton.vue

```diff
@@ -0,0 +1,49 @@
+<script lang="ts">
+import { defineComponent } from "vue";
+
+export default defineComponent({
+	name: "BigAssButton",
+	components: {},
+
+	props: {
+		link: {
+			required: true,
+			type: String,
+		},
+		text: {
+			required: true,
+			type: String,
+		},
+	},
+});
+</script>
+
+<template>
+	<a :href=link>{{ text }}</a>
```

Comment:
**Suggestion:** 
Since you're naming this a "button", I suppose you're gonna use it as a button. For accessibility, it will be read out as a link by VoiceOver. Adding `role="button"` will make it behave as a button
```suggestion
	<a role="button" :href=link>{{ text }}</a>
```

Reply:
So yah, I should have maybe called this something else. It is a button only in name - a link in reality (just looks like a button).